src/main/java/com/faforever/client/main/event/DeleteLocalReplayEvent.java

```diff
@@ -0,0 +1,12 @@
+package com.faforever.client.main.event;
+
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+
+import java.nio.file.Path;
+
+@RequiredArgsConstructor
+public class DeleteLocalReplayEvent {
+  @Getter
+  private final Path replayFile;
```

Comment:
```suggestion
@Value
public class DeleteLocalReplayEvent {
  Path replayFile;
```

Use @ Value here as it does everything you need with one annotation